### ISSUE TITLE
Full size not working. Breaks the camera stream.

### ISSUE THREAD

**Issue by StephaneAdeso** [AlexxIT/WebRTC#645]
## Software version and hardware:

WebRTC Camera version: 3.5.1
HASS version: 2024.1.5
runing on a raspberry pi 4 8gb

## Context:
I have a fresh install of Home Assistant. I installed WebRTC Camera yesterday. Everything works fine, but as soon as you maximize the window of a camera, the stream stop working. It also happens on the Android app when you turn your phone to have the full screen view.

When i open the browser on my computer and press maximize, the erro that apear in the devtools is:
![image](https://github.com/AlexxIT/WebRTC/assets/55595063/40083743-b5c7-4901-b15e-8612d10f04c4)

## How i installed WebRTC:

The way i installed Webrtc is the following:
1. I installed WebRtc from the HACs store.
1. I added the webRTC card directly in my dashboard using the rtc url.

## Aditional info:

I have no Frigate or specific go2rtc installation. i just installed WebRtc. Thats all.
My Camera is a Dahua.
This is a screen shot on how i did it:
![image](https://github.com/AlexxIT/WebRTC/assets/55595063/282b19ac-89f8-485a-ab71-fb99ca4898f5)

**Comment by StephaneAdeso** [comment]
I tried configuring the camera with the `generic camera` integration. It seems to works fine in the pc web browser, but in the android app, it still breaks when you maximize and turn the phone on landscape mode.

**Comment by patrikjohansson87** [comment]
I have the same problem in IOS app

**Comment by koenvo27** [comment]
Any update on this @AlexxIT please?
In the webbrowser on my W11 pc it works fine but in ios I can't open the streams in fullscreen mode..

**Comment by jsamr** [comment]
Same outcome for me; the stream stops after 3 seconds.
![image](https://github.com/AlexxIT/WebRTC/assets/3646758/3b3c9203-5f2c-43ea-b835-3c605949eb1b)

```yml
type: custom:webrtc-camera
url: garage
```

**Comment by AlexxIT** [comment]
I have tested full screen on my Android and iOS. The problem doesn't recur. Maybe someone who has it can fix it.

**Comment by magdogg** [comment]
@AlexxIT Did you try and flip it to landscape? My stream goes black after a few seconds, every time.

**Comment by AlexxIT** [comment]
Yes I did. Android works fine. iOS doesn't flip stream.

**Comment by magdogg** [comment]
@AlexxIT Sorry if I am misunderstanding, but what do you mean iOS doesn't flip stream?

**Comment by AlexxIT** [comment]
In my tests it has portrait view in landscape orientation.

**Comment by j007bond007** [comment]
same issue here - Pixel 8 having issues when I go full screen in the companion app - but in chrome on android on the same device, it works flawlessly, as it does on desktop.

**Comment by andrewindXXL** [comment]
Same issue. Working on desktop fine, but in app fullscreen mode stream stops after several seconds.
WebRTC Camera version: 3.6.1
HASS version: 2025.12.3
Companion app for Android 2025.11.4

But if add in card
ui: true 
and switch in fullscreen via ui icon -  fullscreen stream don't stop in app.
But I don't want to use the UI, it's too inconvenient for my case.

**Comment by felipecrs** [comment]
> same issue here - Pixel 8 having issues when I go full screen in the companion app - but in chrome on android on the same device, it works flawlessly, as it does on desktop.

That actually seems like an issue in the Companion app, not in the card.

**Comment by H4X0-R** [comment]
Hi, this issue still persists. I have it on several different devices (android and windows) on 2 seperate ha installations
Everytime I landscape fullscreen the stream it stops. Fullscreen directly in go2rtc has no problems, its only with the integration.
Are there any updates?

**Comment by felipecrs** [comment]
Can you guys try in Chrome, not in the Companion App?

**Comment by H4X0-R** [comment]
Thanks for the response. On Android Chrome I can't reproduce it but on Chrome running on Windows it occurs every time same as with the companion app.

**Comment by felipecrs** [comment]
Can you record reproducing the issue and share here? I want to try reproducing it too.

**Comment by H4X0-R** [comment]
Okay, so I can reproduce it on Android Chrome as well and I just figured out an solution. 
Enabling "background: true" solves the issue. Once I turned that on, the stream continued flawlessly. Maybe that could be put in the readme or looked into. The problem seems to be solved from my side.

**Comment by felipecrs** [comment]
Yeah, but even if the stream breaks, the card was supposed to reconnect. I think.
